### PR TITLE
[Feature] Add code for listing commands and refactor welcome message

### LINKED ISSUE #11: As a CEG student, I want to see a welcome message.
So that I will feel welcomed to use the product.

### PR DESCRIPTION
**Pull request by wahkia**
Fixes #11 and #12 .

Users are greeted with a welcome message and can list commands now.

### CODE REVIEW

**Review by chewycj — APPROVED** [review]
LGTM!

File: src/main/java/seedu/exchangecoursemapper/ui/UI.java [hunk 1]
```diff
@@ -3,10 +3,23 @@
...
+                 ________ _______ _____   ______ _______ _______ _______\s
+                |  |  |  |    ___|     |_|      |       |   |   |    ___|
+                |  |  |  |    ___|       |   ---|   -   |       |    ___|
+                |________|_______|_______|______|_______|__|_|__|_______|
+                """;
+        System.out.println(banner);
+        System.out.println(greetingMessage);
     }
 
+    /**
```
Comment (louisjoety) [inline src/main/java/seedu/exchangecoursemapper/ui/UI.java:17]: Correct format for JavaDoc comments.

File: src/main/java/seedu/exchangecoursemapper/parser/Parser.java [hunk 2]
```diff
@@ -1,11 +1,13 @@
 package seedu.exchangecoursemapper.parser;
 
+import seedu.exchangecoursemapper.command.ListCommandsCommand;
 import seedu.exchangecoursemapper.command.ListSchoolCommand;
 
 import java.util.Scanner;
 
 public class Parser {
     public static final String LISTINGSCHOOLS = "list schools";
+    public static final String COMMANDS = "commands";
```
Comment (louisjoety) [inline src/main/java/seedu/exchangecoursemapper/parser/Parser.java:10]: Good use of constants for magic strings.

File: src/main/java/seedu/exchangecoursemapper/command/ListCommandsCommand.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package seedu.exchangecoursemapper.command;
+
+public class ListCommandsCommand extends Command {
```
Comment (louisjoety) [inline src/main/java/seedu/exchangecoursemapper/command/ListCommandsCommand.java:3]: Good use of inheritance.

**Pull request merged**